contents/blog/posthog-vs-mixpanel.md

```diff
@@ -0,0 +1,925 @@
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Automate marketing emails</td>
+        </tr>
+        <tr>
+            <td><strong>Intercom</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Automate your product comms</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+### Alerts & Automation
+
+Mixpanel and PostHog both support popular messaging apps and automation platforms, though PostHog also support Discord and the open-source Zapier alternative, n8n.
+
+<div className="overflow-x-auto -mx-5 px-5">
+<table className="w-full mt-4" style="min-width: 600px;">
+    <thead>
+        <tr>
+            <td className="w-3/12"></td>
+            <td><strong>Mixpanel</strong></td>
+            <td><strong>PostHog</strong></td>
+            <td></td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><strong>Slack</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Send alerts and reports to Slack</td>
+        </tr>
+        <tr>
+            <td><strong>Discord</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Send alerts and reports to Discord</td>
+        </tr>
+        <tr>
+            <td><strong>Microsoft Teams</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Send alerts and reports to Teams</td>
+        </tr>
+        <tr>
+            <td><strong>n8n</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Open source workflow automation</td>
+        </tr>
+        <tr>
+            <td><strong>Twilio</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Automate SMS alerts on events</td>
+        </tr>
+        <tr>
+            <td><strong>Zapier</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Automate workflows on events</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+## Frequently asked questions
```

Comment:
Haven't finished this yet, but interested in any common questions that come up (on Slack, demos etc.) that we should cover here.

Reply:
- As PostHog is for devs, does it take more time to set up: https://posthogusers.slack.com/archives/C01GLBKHKQT/p1665087867384459
- How do you migrate historical data from Mixpanel to PostHog: https://posthogusers.slack.com/archives/CTLTM70RM/p1660820207822279
- Lots of comments on how funnels are different (not really a general question though)
- How do either of these compare to GA: https://posthogusers.slack.com/archives/CT7HXDEG3/p1632178283215200?thread_ts=1632132507.199900&cid=CT7HXDEG3